apps/web/src/app/(dashboard)/documents/data-table-action-dropdown.tsx

```diff
@@ -57,12 +57,16 @@ export const DataTableActionDropdown = ({ row }: DataTableActionDropdownProps) =
   const recipient = row.Recipient.find((recipient) => recipient.email === session.user.email);
 
   const isOwner = row.User.id === session.user.id;
+  const isOwnerARecipient = row.Recipient.some(
+    (recipient) => recipient.email === session.user.email,
+  );
```

Comment:
Just noting there's a variable `recipient` (not a good name) a few lines up which is effectively the same thing